cedar-policy/CHANGELOG.md

```diff
@@ -4,28 +4,28 @@
 
 ### Added
 
-- Export `entities::EntitiesError` error type.
 - New methods exported for `EntityTypeName`.
   - `basename` to get the basename (without namespaces).
   - `namespace_components` to get the namespace as an iterator over its components.
   - `namespace` to get the namespace as a single string.
-- Fixed bug (#150) around implicit namespaces in action definitions.
 
 ### Changed
-- When an entity type shape or action context is declared with type other
-  than `Record`, the error message will indicate the affected entity type or
-  action.
-- Update how record types are treated by the validator to support "open" and
-  "closed" record types.  Record types written in schema are now closed. In
-  particular, this applies to the action context, so `context has attr` can now
-  have type False where before it had type Boolean, creating some new
-  short-circuiting opportunities.  The same applies to record literals.
```

Comment:
I don't think we want to delete this -- it's still an unreleased change.

Alternatively: maybe we want to delete all the "unreleased changes" from the the release CHANGELOG? It's a little weird to include them since they're not in this branch.

Reply:
In a hypothetical world where the changelog update always perfectly happened in the PR responsible for the change, that would happen automatically.  (Changes we haven't cherry-picked to this branch wouldn't appear in the CHANGELOG for this branch either.)

Reply:
Deleted the unreleased section.